Week01/Problem04/apavlova_04.py

```diff
@@ -0,0 +1,23 @@
+#this is very brute force but I cant come up with something better
```

Comment:
My solution may be brute force too, but I cast the value to a string to check its reversal:
```
def check_palindrome(val):
    strval = str(val)
    if strval == strval[::-1]:
        return True
    else:
        return False
```